### PR TITLE
Flax develop

### PR DESCRIPTION
**Pull request by KamenDimitrov97**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,25 +36,27 @@ deps: ## Installs dependencies
...
-	poetry run pytest tests/api
+test-component: ## runs component tests
+	poetry run pytest -v tests/api
 
-format: deps ## Formats your code automatically.
-	poetry run isort .
-	poetry run black .
+
+unit: ## runs component tests
+	poetry run pytest -v tests/unit
```
Comment (lindenmckenzie) [inline Makefile:59]: Please can this be changed back to 'test' for consistency.

Comment (lindenmckenzie) [reply Makefile:59]: also the comment is inaccurate.

File: .env.default [hunk 2]
```diff
@@ -0,0 +1,9 @@
+BERLN_API_HOST=0.0.0.0
+BERLN_API_PORT=28900
+
+BERLIN_API_NAMESPACE=dp_nlp_berlin_api
```
Comment (lindenmckenzie) [inline .env.default:4]: This is different to `BERLIN_API_LOGGING_NAMESPACE` and `LOGGING_NAMESPACE` in other files.

File: swagger.yaml [hunk 3]
```diff
@@ -9,6 +9,56 @@ info:
...
+                subdiv:
+                  - "on"
+                  - "ontario"
+              london_us:
+                encoding:	"UN-LOCODE"
+                id:	"us:ldn"
+                key:	"UN-LOCODE-us:ldn"
+                words:	
+                  -	"london"
+                names:
```
Comment (lindenmckenzie) [inline swagger.yaml:47]: Spacing a bit odd here.

**Review by lindenmckenzie — CHANGES_REQUESTED** [review]
Some changes listed above - additionally, the swagger spec does not parse with https://editor.swagger.io/

File: Makefile (hunk 1 above)
Comment (KamenDimitrov97) [reply Makefile:59]: Yeah I'll fix that but I changed it to unit so that it's consistent with category api

File: .env.default (hunk 2 above)
Comment (KamenDimitrov97) [reply .env.default:4]: Not sure what you mean, BERLIN_API is just the prefix for dynaconf. 
So you set what the prefix will be and that instance of dynaconf will take only env variables that start with BERLIN_API
and you get them by querying for the env var without the prefix e.g. LOGGING_NAMESPACE
if that's what you meant tho

Comment (lindenmckenzie) [reply .env.default:4]: so here it is "NAMESPACE"
elsewhere it is "LOGGING_NAMESPACE"

File: app/logger.py [hunk 4]
```diff
@@ -17,6 +17,6 @@ def configure_logging():
 
 def setup_logger():
     return structlog.get_logger(
-        namespace=NAMESPACE,
+        namespace=settings.NAMESPACE,
```
Comment (lindenmckenzie) [inline app/logger.py:20]: isn't it `LOGGING_NAMESPACE`

Comment (lindenmckenzie) [reply app/logger.py:20]: oh I see it now.

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**